types/meteor/mongo.d.ts

```diff
@@ -318,9 +371,30 @@ declare module 'meteor/mongo' {
                 numberAffected?: number | undefined;
                 insertedId?: string | undefined;
             };
+            /**
+             * Modify one or more documents in the collection, or insert one if no matching documents were found. Returns an object with keys `numberAffected` (the number of documents modified) and
+             * `insertedId` (the unique _id of the document that was inserted, if any).
+             * @param selector Specifies which documents to modify
+             * @param modifier Specifies how to modify the documents
+             * @param callback If present, called with an error object as the first argument and, if no error, the number of affected documents as the second.
+             */
+            upsertAsync(
+                selector: Selector<T> | ObjectID | string,
+                modifier: Modifier<T>,
+                options?: {
+                    /** True to modify all matching documents; false to only modify one of the matching documents (the default). */
+                    multi?: boolean | undefined;
+                },
+                callback?: Function,
+            ): Promise<{
+                numberAffected?: number | undefined;
+                insertedId?: string | undefined;
+            }>;
+            _createCappedCollection(byteSize?: number, maxDocuments?: number): void;
```

Comment:
This is prefixed with `_` - should this also be deprecated? (same for `globals/mongo.d.ts`)

Reply:
I wouldn't do it just yet, as `createCappedCollectionAsync` is still somehow experimental.